packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,6 +42,22 @@ bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
+std::string Base64EncodeUri(const std::string& prefix,
+                            const std::string& data) {
+    std::string data_base64;
+    base::Base64Encode(data, &data_base64);
```

Comment:
No, it should apply to Widevine and identity key systems as well. It is probably better to change it in a separate patch, so I think it is fine to leave it as is in this patch. Feel free to submit another pull request on this if you want; if not, we'll address it :)